### ISSUE TITLE
Low master/hitsound volume level causes crashes

### ISSUE THREAD

**Issue by yugecin** [itdelatrisu/opsu#306]
**Version:** 0.15.0
**Build date:** 2017-05-29 02:26
**OS:** Windows 7 (amd64)
**JRE:** 1.8.0_60
**OpenGL version:** 4.5.13453 Compatibility Profile Context 21.19.134.1 (ATI Technologies Inc.)
**Stack trace:**
```
java.lang.IllegalArgumentException: Requested value -84.43697 smaller than allowable minimum value -80.0.
	at javax.sound.sampled.FloatControl.setValue(FloatControl.java:222)
	at com.sun.media.sound.DirectAudioDevice$DirectDL$Gain.setValue(DirectAudioDevice.java:853)
	at itdelatrisu.opsu.audio.MultiClip.start(MultiClip.java:143)
	at itdelatrisu.opsu.audio.SoundController.playClip(SoundController.java:308)
	at itdelatrisu.opsu.audio.SoundController.playHitSound(SoundController.java:340)
	at itdelatrisu.opsu.GameData.sendSliderTickResult(GameData.java:1446)
	at itdelatrisu.opsu.objects.Slider.mousePressed(Slider.java:606)
	at itdelatrisu.opsu.states.Game.sendGameKeyPress(Game.java:2256)
	at itdelatrisu.opsu.states.Game.runReplayFrame(Game.java:2218)
	at itdelatrisu.opsu.states.Game.addReplayFrameAndRun(Game.java:2205)
	at itdelatrisu.opsu.states.Game.gameKeyPressed(Game.java:1459)
	at itdelatrisu.opsu.states.Game.keyPressed(Game.java:1226)
	at org.newdawn.slick.state.StateBasedGame.keyPressed(StateBasedGame.java:445)
	at org.newdawn.slick.Input.poll(Input.java:1197)
	at org.newdawn.slick.GameContainer.updateAndRender(GameContainer.java:680)
	at itdelatrisu.opsu.Container.gameLoop(Container.java:136)
	at itdelatrisu.opsu.Container.start(Container.java:80)
	at itdelatrisu.opsu.Opsu.main(Opsu.java:213)
```

~~Happens on this map (any dif): https://osu.ppy.sh/b/19843~~
Edit: not caused by that map, it's caused by the volume setting.
Generally, a very low master volume level (or hitsound level maybe too), results in this exception.
This should help in reproducing the crash:
![image](https://user-images.githubusercontent.com/12662260/27657155-f5043572-5c4b-11e7-88ac-0b9656465e52.png)

**Comment by tpenguinltg** [comment]
Does this happen randomly? I tried it with these volume settings and it didn't crash for me. It might be a Windows thing.

**Comment by yugecin** [comment]
I guess it only happens on some maps that change the hitsound volume per object. It crashes as soon as I click on an object, the map I linked previously is guaranteed to crash for me.

**Comment by tpenguinltg** [comment]
I tried it with the map and it seems fine for me, which suggests to me that it's a Windows thing.

**Comment by yugecin** [comment]
Seems like it is caused by my skin.
To reproduce, use a dummy `normal-sliderwhistle.wav` (0 bytes) file.
It still looks like a Windows only problem though, no problems on arch or mac.

**Comment by itdelatrisu** [comment]
Fixed in 212a395, let me know if this keeps happening.
